camping.gemspec

```diff
@@ -2,38 +2,39 @@
 # camping_spec
 
 NAME = "camping"
-BRANCH = "2.3"
+BRANCH = "3.0.0"
 GIT = ENV['GIT'] || "git"
 REV = `#{GIT} rev-list HEAD`.strip.split.length
-VERS = ENV['VERSION'] || (REV.zero? ? BRANCH : [BRANCH, REV] * '.')
+VERS = ENV['VERSION'] || BRANCH
 
 RDOC_OPTS = ["--line-numbers", "--quiet", "--main", "README"]
 
-def camping_spec
-  @spec ||= Gem::Specification.new do |s|
-    s.name = NAME
-    s.version = VERS
-    s.platform = Gem::Platform::RUBY
-    # s.extra_rdoc_files = FileList["README.md", "CHANGELOG", "COPYING", "book/*"].to_a
-    s.rdoc_options += RDOC_OPTS + ['--exclude', '^(examples|extras)\/', '--exclude', 'lib/camping.rb']
-    s.summary = "miniature rails for anyone"
-    s.author = "why the lucky stiff"
-    s.email = 'why@ruby-lang.org'
-    s.homepage = 'http://camping.rubyforge.org/'
-    s.executables = ['camping']
-    s.add_dependency('rack', '>=1.0')
-    s.add_dependency('mab', '>=0.0.3')
-    s.required_ruby_version = '>= 1.8.2'
+@spec ||= Gem::Specification.new do |s|
+  s.name = NAME
+  s.version = VERS
+  s.platform = Gem::Platform::RUBY
+  # s.extra_rdoc_files = FileList["README.md", "CHANGELOG", "COPYING", "book/*"].to_a
```

Comment:
Can this be removed?

Reply:
I'm thinking yes? Maybe we'll want to ad Yardoc back in the future but for now we can delete.